Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.WebContent.sb.in

```diff
@@ -930,7 +930,9 @@
             "com.apple.iphone.axserver-systemwide"
             "com.apple.mobileassetd.v2"
             "com.apple.mobilegestalt.xpc"
+#if !ENABLE(GPU_PROCESS_BY_DEFAULT)
             "com.apple.tccd"
+#endif
```

Comment:
Can you also remove the `plistbuddy` commands from `process-entitlements.sh`?

Reply:
It looks like Speech Recognition might want to use the microphone in the WebContent process. This should probably be moved to the GPU process (and perhaps already has been).

See `SpeechRecognitionRealtimeMediaSourceManager` for details.

Reply:
> Can you also remove the `plistbuddy` commands from `process-entitlements.sh`?

Ah, good point, will fix!

Thanks for reviewing!

Reply:
> It looks like Speech Recognition might want to use the microphone in the WebContent process. This should probably be moved to the GPU process (and perhaps already has been).
> 
> See `SpeechRecognitionRealtimeMediaSourceManager` for details.

That is a good point. I have removed the sandbox extensions for this case in the latest patch.